README.md

```diff
@@ -31,11 +31,11 @@ metadata such as the number of words and named entities from the text.
 >>> from textpipe import doc, pipeline
 >>> sample_text = 'Sample text! <!DOCTYPE>'
 >>> document = doc.Doc(sample_text)
->>> print(document.clean_text)
+>>> print(document.clean_text())
```

Comment:
we're completely stepping away from properties?

Reply:
I'm not sure whether this is a good decision, but given the fact that we want the possibility to map operations to models, we need to be able to pass arguments. This seemed like the most general solution for that.

Reply:
Right. Alternatively it would be a property `cleaned_text` and a method `clean_text()`

Reply:
Did you see that there is already a PR for keeping both with #53?